system/mp/mpcomm.cpp

```diff
@@ -2126,10 +2141,10 @@ int CMPConnectThread::run()
 #if defined(_USE_OPENSSL)
                 if (parent->useTLS)
                 {
+                    Owned<ISecureSocket> ssock = secureContextServer->createSecureSocket(sock.getClear());
                     int tlsTraceLevel = SSLogMin;
-                    if (parent->mpTraceLevel)
+                    if (parent->mpTraceLevel > DebugMsgThreshold)
```

Comment:
Should this be testing if > MyVerboseLevel (which is higher than the default for now), to suppress extra tracing?